lib/svtplay_dl/fetcher/dash.py

```diff
@@ -25,43 +25,60 @@ def __init__(self, url):
             url, "This is a live DASH stream, and they are not supported.")
 
 
-def templateelemt(element, filename, idnumber):
+def templateelemt(element, filename, idnumber, offset_sec, duration_sec):
     files = []
+    timescale = 1
+    duration = 1
+    total = 1
 
     init = element.attrib["initialization"]
     media = element.attrib["media"]
     if "startNumber" in element.attrib:
         start = int(element.attrib["startNumber"])
     else:
-        start = 0
+        start = 1
```

Comment:
why change this number to `1`?

Reply:
I noticed that the Default in MPD files usually was 1.

Reply:
ahh true. :)